inbound.go

```diff
@@ -62,7 +64,8 @@ func (c *Connection) handleCallReq(frame *Frame) bool {
 
 	call := new(InboundCall)
 	call.conn = c
-	ctx, cancel := newIncomingContext(call, callReq.TimeToLive, &callReq.Tracing)
+	// TODO used to propagate tracingDisabled
```

Comment:
think it was a note to myself, I'll remove it. The disabling is propagated and covered by the unit test.